tachyon/math/finite_fields/baby_bear/packed_baby_bear_avx2.cc

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2022 The Plonky3 Authors
+// Use of this source code is governed by a MIT/Apache-2.0 style license that
+// can be found in the LICENSE-MIT.plonky3 and the LICENCE-APACHE.plonky3
+// file.
+
+#include "tachyon/math/finite_fields/baby_bear/packed_baby_bear_avx2.h"
+
+#include <immintrin.h>
+
+#include "tachyon/math/finite_fields/packed_prime_field32_avx2.h"
+
+namespace tachyon::math {
+
+namespace {
+
+__m256i kP;
+__m256i kInv;
+__m256i kZero;
+__m256i kOne;
+
+__m256i ToVector(const PackedBabyBearAVX2& packed) {
+  return _mm256_loadu_si256(
+      reinterpret_cast<const __m256i_u*>(packed.values().data()));
+}
+
+PackedBabyBearAVX2 FromVector(__m256i vector) {
+  PackedBabyBearAVX2 ret;
+  _mm256_storeu_si256(reinterpret_cast<__m256i_u*>(ret.values().data()),
+                      vector);
+  return ret;
+}
+
+__m256i Add(__m256i lhs, __m256i rhs) { return AddMod32(lhs, rhs, kP); }
+
+__m256i Sub(__m256i lhs, __m256i rhs) { return SubMod32(lhs, rhs, kP); }
+
+__m256i Neg(__m256i val) { return NegMod32(val, kP); }
```

Comment:
I thought we decided to use full name for unary operations. If so, please change it to `Negate`

Reply:
Oh got it, that makes sense!